pyrobosim/pyrobosim/core/robot.py

```diff
@@ -23,6 +24,10 @@ def __init__(
         radius=0.0,
         height=0.0,
         color=(0.8, 0.0, 0.8),
+        max_linear_velocity=np.inf,
```

Comment:
I am in favour of always getting a limit from user instead of an infinity default limit.

Reply:
I think the default should actually be unlimited, because there will probably be a core set of users who just want to command a velocity based on something else. So I might leave this as is.